packages/server/migrations/201907100048002-templates.js

```diff
@@ -0,0 +1,41 @@
+module.exports = {
+  up: async function (queryInterface, Sequelize) {
+    try {
+      await queryInterface.createTable('Templates', {
```

Comment:
Can you be more specific than “Template”? Is this a “NotificationTemplate” or an “EmailTemplate”?

Reply:
It is not only for "notification" or "emails" templates, it is common way of storing "system templates", do not connect it with "notifications" or "emails".